sdk/storage/azure-storage-common/src/crypt.cpp

```diff
@@ -1,15 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "azure/storage/common/crypt.hpp"
 #include <azure/core/platform.hpp>
```

Comment:
#include ""

Reply:
?

Reply:
https://azure.github.io/azure-sdk/cpp_design.html#physical-design

```
YOU SHOULD include files using quotes (“) for files within the same git repository, and angle brackets (<>) for external dependencies.
```

Reply:
We need to update that guideline.
The "" are used within the same repo, yes, but for unique packages. The <> should means that the headers is comming externally.
The Azure Storage blobs should consider Azure Core as an external including. Even Azure Storage Common should be considered external (because it's shipped in a different package).

Anyhow, this is not the intention of this PR to fix the `include` usage in the code, but only the formatting on it.

The `include` fixes would require a different PR

Reply:
I will create another PR fixing the `include`s

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1211